netplotbrain/plot.py

```diff
@@ -223,8 +220,41 @@ def plot(nodes=None, fig: Optional[plt.Figure] = None, ax=None, view: str = 'L',
                 _plot_springlayout(ax, nodes=nodes, edges=edges, node_color=node_color, node_size=node_size,
                                    edge_color=edge_color, edge_weights=edge_weights, highlight_nodes=highlight_nodes, **profile)
             ax.view_init(azim=azim[fi], elev=elev[fi])
-
-            _add_subplot_title(ax, azim[fi], elev[fi], subtitle_frame, hemi_frame, viewtype[fi], **profile)
+                       
```

Comment:
This needs to be made more concise. 

I only see only option needed:

if there is no input for title or subtitle... then title is 'auto' and subtitles is None (or maybe it should be 'auto' too)

When then occurs, set title to None and subtitles to auto. Why are any other automatic scenario needed other than this.